src/runtime/basic-runtime.ts

```diff
@@ -189,6 +189,19 @@ export class BasicRuntime implements BasicSubscriptions {
     return this.configuredPromise_;
   }
 
+  private writePageConfig_(markupValues: {
+    type: string | Array<string>;
+    isAccessibleForFree: boolean;
+    isPartOfType: string | Array<string>;
+    isPartOfProductId: string;
+  }): void {
+    this.pageConfigWriter_ = new PageConfigWriter(this.doc_);
+    this.pageConfigWriter_.writeConfigWhenReady(markupValues).then(() => {
```

Comment:
Could we flatten this with async/await?
```suggestion
    await this.pageConfigWriter_.writeConfigWhenReady(markupValues);
```